src/components/Badge/style.css

```diff
@@ -24,13 +24,13 @@
 }
 
 .notify {
-  background-color: var(--rvr-color-notify);
+  background-color: var(--rvr-color-notification);
   color: var(--rvr-white);
```

Comment:
You're right about the `rvr-color` namespace.
I also agree that it's weird intermixing non-semantics and semantic colors, e.g. `--rvr-white` and `--rvr-color-notify`.

For now, I didn't add consistent color ramps (with `text`, `main`, `hover`, etc.) for most of the semantics, because the distinction isn't there in the Figma library.

Would you be ok with leaving it as-is for this iteration? I'll work on adding more consistent color ramps for each semantic color in the future.